tests/lib.rs

```diff
@@ -0,0 +1,16 @@
+extern crate libftd2xx_ffi;
+
+use libftd2xx_ffi::*;
```

Comment:
```suggestion
use libftd2xx_ffi::FT_GetLibraryVersion;
```

A direct import is more clear here.